code-challenges.test.js

```diff
@@ -14,19 +16,70 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// PSEUDOCODE
+// Use red-green refactor process to write test for the given prompt
+// create fxn called fibFxn
+// input: number
+// output: array of input length filled with Fibonacci sequence
+// initialize empty array. set first num to 0, second num to 1
+// iterate through array using for loop with starting index 0 and stop index at < inputted number + 1
```

Comment:
I really like this pseudo! I would like to see next week's pseudo look like this one.